src/components/routes/groups/api/eventsApi.ts

```
@@ -0,0 +1,414 @@
+): Promise<FkOption> {
+  const PAGE = 20;
+  const MAX_PAGES = 5;
+  try {
+    for (let page = 0; page < MAX_PAGES; page++) {
+      const res = await searchFn({ skip: page * PAGE, limit: PAGE });
+      const match = res.items.find((item) => item.id === id);
+      if (match) return match;
+      const fetched = res.skip + res.items.length;
+      if (fetched >= res.total || res.items.length === 0) break;
+    }
+  } catch {}
```

ESLint

[error] Empty block statement.

(no-empty)